toolboxes/feel/feelmodels/modelvf/fluidmecstresstensor.hpp

```diff
@@ -371,14 +371,14 @@ class FluidMecStressTensorImpl : public Feel::vf::ExprDynamicBase
         value_type
         evaliq( uint16_type i, uint16_type c1, uint16_type c2, uint16_type q ) const override
         {
-            if constexpr ( this_type::specific_expr_type::value != ExprApplyType::EVAL )
+            if constexpr ( !this_type::is_applied_as_eval )
                  CHECK( false ) << "not allow";
```

Comment:
could you put a more explicit message:
Only test and trial expressions are allowed, evaluating the expression at the quadrature points is forbidden

Reply:
or something like that